### PR TITLE
Fix boolean parsing

### PR DESCRIPTION
**Pull request by alex-deaconu**
get() for boolean false returns null
Interpret null as false

### CODE REVIEW

**Comment by alex-deaconu** [comment]
The JSONParser was interpreting `"true"` and `"false"` strings correctly. However, `true`, `false` boolean values were not used until I started testing w/ the new apps.

@olegrise please review. Thanks.

File: src/com/risevision/common/client/json/JSOModel.java [hunk 1]
```diff
@@ -200,7 +200,7 @@ public final boolean getBoolean(String key, Boolean defaultValue) {
 			String value = get(key);
 			if (Boolean.TRUE.toString().equals(value))
 				return true;
-			else if (Boolean.FALSE.toString().equals(value))
+			else if (value == null || Boolean.FALSE.toString().equals(value))
```
Comment (olegrise) [inline src/com/risevision/common/client/json/JSOModel.java]: @alex-deaconu to be honest, the logic looks strange to me. If `value == null`, then I would expect that  `defaultValue` is returned. No?

**Comment by alex-deaconu** [comment]
@olegrise you're right. I've modified the parser to use JSNI to directly get the boolean value from the object. Let me know if this new version improves the situation.

Thanks!

**Comment by olegrise** [comment]
Looks good :+1:

**Pull request merged**